### DISCUSSION TITLE
Biostar_plot_heatmap.r (There was an error opening this document. The file is damaged and could not be repaired)

### DISCUSSION THREAD

**Discussion by mohannadmd** [biostars/biostar-handbook-issues#263] [Q&A]
I generated this Q&A in case someone else faced the same issue in opening the heatmap.pdf file using adobe-acrobat reader.

How to fix it:

Uncomment the last command below i.e (remove the #, before dev.off), and rerun the r.script

# #dev.off()

Cheers,
MD

**Comment by ialbert** [comment]
this has been now fixed in the latest update to the code,

thanks for reporting
